CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+Please open a new issue or new pull request for bugs, feedback, or new features you would like to see. If there is an issue you would like to work on, please leave a comment and we will be happy to assist. New contributions and contributors are very welcome!
+
+New to github or open source projects? If you are unsure about where to start or haven't used github before, please feel free to contact the package maintainers. Want more information about how to make a contribution? 
```

Comment:
do we have a general place to point people to for contributing or should we leave it to each package to add it to their documentation?

Reply:
I think it would be good to have some general contributing docs.  Each package can then either simply link to those or use them as a starting point for their own custom contributing docs.

Reply:
@larrybradley you wanna draw up some contributing docs? we can point to them whenever they are ready?  Probably something close to what astropy has is reasonable. We'll need something different for the mission packages that need science team approval for code changes.